core/storage/btree.rs

```diff
@@ -1424,6 +1543,11 @@ impl Cursor for BTreeCursor {
         }
     }
 
+    fn last(&mut self) -> Result<CursorResult<()>> {
+        self.move_to_rightmost()?;
```

Comment:
This can return a `CursorResult::IO` right, so we should yield (return) here in that case?